### ISSUE TITLE
Support for graph themes

### ISSUE THREAD

**Issue by nickerso** [opencor/opencor#1486]
Now that graphs are so visually configurable, it would be convenient to have some pre-defined themes that could be chosen that would set the "default" values for background and foreground colours, an array of compatible line colours that would automatically step through as curves added, etc. For example, right click a graph panel and easily choose "black and white" or "fluro with black background" :)

**Comment by agarny** [comment]
I agree that it would be nice to have, and wouldn't be that difficult to implement (I think! :)).
